change/@uifabric-styling-2019-07-17-15-13-36-iconsReact.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Removing react dependency in @uifabric/styling library.",
+  "type": "patch",
```

Comment:
Would removing a dependency of a package count as `patch` only? I would go on the safe side and do a minor, but would let you decide. (@kenotron any thoughts?)

Reply:
Removing dependency like this should be fine as patch